### ISSUE TITLE
Is it possible to synchronize via Windows with SyncFTP?

### ISSUE THREAD

**Issue by rtulke** [alex-donnan/SyncFTP#7]
Hi I have several devices under Mac I have got SyncFTP running. Now I wanted to set it up on Windows. But here I get error messages.  What do I have to do to be able to synchronize with SFTP under Windows?

**Comment by alex-donnan** [comment]
What errors are you receiving? I tested on Mac and Windows Obsidian versions so there's a chance it's the SFTP config.

**Comment by alex-donnan** [comment]
Also are they both pointing to the same SFTP?

**Comment by alex-donnan** [comment]
@rtulke Just wanted to follow up with this issue.

**Comment by rtulke** [comment]
Sorry for late response ![image](https://github.com/alex-donnan/SyncFTP/assets/620092/16ea0f5f-b28a-43ef-a999-04dfcfbda345) here a screenshot

**Comment by rtulke** [comment]
Yes both connections windows/Mac with same credentials and connection settings

**Comment by zellio22** [comment]
![image](https://github.com/alex-donnan/SyncFTP/assets/49569121/4e57748c-3fa4-44aa-b0b7-08ed28d72581)
i have same message

**Comment by merlinuwe** [comment]
Thanks for your plugin. 

I have the same issue.

![image](https://github.com/alex-donnan/SyncFTP/assets/4138869/2718a1f1-78b3-405c-846c-344335312a1a)

(The connection and uploading with filezilla works,)

**Comment by backd00rbenny** [comment]
Having the same problem, except I'm running Arch linux.

**Comment by BartRoozen** [comment]
Error: connect->getConnection: Timed out while waiting for handshake
Failed to connect to SFTP: TypeERror: fileObjects not iterable

Does this mean my credentials are correct, or not?

**Comment by nixxizlab** [comment]
Same problem. Obsidian from flatpak.

**Comment by jeanpantaleon** [comment]
To y'all, it's a bug, and there're a lot in his code, he will not fix it i think

**Comment by Utro-tvar** [comment]
I guess I fixed this problem:
https://github.com/Utro-tvar/SyncFTP
